config/downloads.yml

```diff
@@ -11,3 +11,15 @@ sample_unitid:
   ead:
     href: 'http://example.com/sample.xml'
     size: 123456
+
+umich-bhl-0234:
+  ead:
+    href: 'https://github.com/sul-dlss/arclight-demo/raw/master/data/ead/umich-bhl/umich-bhl-0234.xml'
+    size: 91781
+2016C32-xml:
```

Comment:
there is a typo here -- it should be just `2016C32` without the `-xml`. this is the <unitid> id value as-is.

```xml
            <unitid label="Collection Number" countrycode="US" repositorycode="CSt-H">2016C32</unitid>
```